CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,77 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Author(s):
+    kwattorama <61426179+kwattorama@users.noreply.github.com>
+    xames3 <44119552+xames3@users.noreply.github.com>
+-->
+
+<!-- markdownlint-disable MD033 MD041 -->
+![Code of Conduct](https://github.com/xames3/kaamiki/blob/assets/elements/img/code-of-conduct-banner.png?raw=true)
+<h1><p align="center">Our Code of Conduct</p></h1>
+
+We want everyone to progress in an ecosystem that empowers developers and users alike to reach their potential whilst encouraging self-growth and effective collaboration. In short, a space that is safe for all. A space which will benefit everyone that participates in the process. A space which encourages new developers to learn and enter in the world of programming.
+
+In an effort to create such place, we are building our Code of Conduct.<br>We do not publish this because we fear an inappropriate behavior, but because we appreciate and love the extraordinary respect and values of our team and would like every contributor to respect it as well. This is through discussion and collaboration that we grow, and with our growth we improve.
+
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, harassment-free and healthy environment for everyone, regardless of academic education and years of experience.
+
+As community members, contributors and maintainers of this project, we are thankful and commit to respect all people who participate in the community through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+## Our standards
+
+- We welcome and encourage participation by everyone and are committed to being in a community where everyone feels good about joining.
+- We sacredly believe in the concept of **[Vasudhaiv Kutumbkam](https://en.wikipedia.org/wiki/Vasudhaiva_Kutumbakam)** (The World is One Family) in letter and spirit. Although we may not be able to satisfy everyone, we will always work to treat everyone well.
+- We don't encourage support and suggestions on demand, we expect everyone to be proactive. We believe in proactive help or contribution to a problem. While doing so, please be careful not to be too patronizing or disrespectful. When someone turns to you for help, always be open, listen and try to put yourself in the position of your counterpart. If you don't have time, let the person know or maybe refer to a team member who is just as familiar with the topic.
```

Comment:
this point will relate more in the expected behavior section

Reply:
Rearranged [here](https://github.com/kaamiki/kaamiki/commit/70d1aad8ee5bcf65a38d389ef53fb58e4c9508fc).